project/plugins.sbt

```diff
@@ -32,3 +32,5 @@ concurrentRestrictions in Global := Seq(
 )
 
 addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.3.3")
+
+scalaVersion := "2.12.7"
```

Comment:
https://github.com/scala/scala/blob/c277d7d7ca79d0f66bb458bcfb03c060b77b9357/project/plugins.sbt#L2

We should add `scalaVersion` setting due to `"-Xfatal-warnings"`

Reply:
Why? Which Scala version does sbt use otherwise?

Reply:
sbt 1.2.3 use Scala 2.12.6
- https://github.com/scala/scala/blob/v2.12.6/src/library/scala/collection/immutable/StringLike.scala#L136-L137
- https://github.com/scala/scala/blob/v2.12.7/src/library/scala/collection/immutable/StringLike.scala#L136-L137
- https://github.com/scala/scala/commit/46db85852564f123697666da24ef216c5b2b29b3

Reply:
Thank you!